src/common/lib/polkadot-provider/query/genetic-analyst-qualifications.js

```diff
@@ -0,0 +1,5 @@
+export async function queryGeneticAnalystQualifications(api, id) {
+  let res = (await api.query.geneticAnalystQualifications.geneticAnalystQualifications(id))
```

Comment:
why are you using let instead of const and (? please update with better and consistent practice